static/components/product-card/product-card.html

```diff
@@ -1,6 +1,6 @@
 <q-card class="card--product">
   <q-img
-    :src="product.images ? product.images[0] : '/nostrmarket/static/images/placeholder.png'"
+    :src="product.images.length > 0 ? product.images[0] : '/nostrmarket/static/images/placeholder.png'"
```

Comment:
- we should check if `product.images` is `null/undefined` (it might not always be an empty list
- `:src="(product.images && product.images.length) > 0 ? product.images[0] :`